icicle/src/ntt/tests/verification.cu

```diff
@@ -1,20 +1,19 @@
 
-#define CURVE_ID BLS12_381
+#define CURVE_ID 1
 
-#include "primitives/field.cuh"
-#include "primitives/projective.cuh"
+#include "curves/bn254/bn254.cuh"
```

Comment:
why directly include and not via curve_config? and BTW I think it's broken since there is no such file